lib/msf/core/post/file.rb

```diff
@@ -741,8 +889,6 @@ def get_drives
     drives
   end
 
-protected
-
```

Comment:
Was there a reason to remove protected status from methods below this?

Reply:
Uhhh no, I'm not sure how that got in there but let me add that back in, that shouldn't have happened.

Reply:
@jmartin-r7 I think this may have been a case of the `protected` method moving up in the code. if you look up further in the file you can see that we have three methods above this line all which are defined as `protected` so I don't think there is a reason to redeclare things again as protected here?

Reply:
Understood, I see it [here](https://github.com/rapid7/metasploit-framework/pull/15987/files#diff-cc6585bd12e8d43e126c2e1a79e4132f56c1e180483f410ccbf90cfb04c5e5c8R759), looks like it was double defined in the original.